lib/oauth.js

```diff
@@ -131,6 +131,10 @@ const authClient = new Model({
             ['X-CSRF-Token']: token
           };
 
+          if (this._tokenDetails && this._tokenDetails.access_token) {
+            deleteHeaders['Authorization'] = 'Bearer ' + this._tokenDetails.access_token
+          }
```

Comment:
Similar comment here. This code is absolutely fine but if the token details are missing, then you aren't logged in anway (I think.)